region/maridia/outer.json

```diff
@@ -1128,6 +1125,23 @@
                 ]}
               ],
               "unlock": null
+            },
+            {
+              "id": 11,
+              "requires": [
+                {"or": [
+                  "canSwim",
+                  "canSuiitlessMaridia"
+                ]},
+                {"canComeInCharged":{
+                  "fromNode": 2,
+                  "framesRemaining": 0,
+                  "shinesparkFrames": 60
+                }}
+                ]}
+              ],
+              "unlock": null,
+              "note": "One-way link for shinesparking."
```

Comment:
Seems to me like it would be better to make a spark link directly to 1, since you can do that with a diagonal spark and 1 to 11 is free (while 11 to 1 has reqs)

Reply:
Good point. Time to fire up paint again.